utils/config-utils.sh

```diff
@@ -133,7 +152,7 @@ config::await_service_running() {
 
     docker service rm instant_await-helper &>/dev/null
 
-    try "docker stack deploy -c $await_helper_file_path instant" "Failed to deploy await helper"
+    try "docker stack deploy -c $await_helper_file_path instant" catch "Failed to deploy await helper"
```

Comment:
Should this not be a try-throw? If the await helper fails to deploy, we could have a scenario where config importers run too quickly, making it a fairly significant deploy failure.

Reply:
yes right, fixed in [a0370df](https://github.com/jembi/platform/pull/188/commits/a0370dfd3a72d6c29352623304555257bccbe370)